stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -678,34 +678,66 @@ public final class _DataStorage {
     }
     
     @inline(__always)
-    public func mutableCopy() -> _DataStorage {
+    public func mutableCopy(_ range: Range<Int>) -> _DataStorage {
```

Comment:
Can check for the swift case in the inline, but call out to a non-inlined function for the rest?